subsys/net/ip/Kconfig

```diff
@@ -279,6 +279,22 @@ config NET_TC_NUM_PRIORITIES
 	default NUM_COOP_PRIORITIES if NET_TC_THREAD_COOPERATIVE
 	default NUM_PREEMPT_PRIORITIES if NET_TC_THREAD_PREEMPTIVE
 
+config NET_TC_THREAD_PRIO_CUSTOM
+	bool "Net threads priority custom"
+	default n
```

Comment:
Options are "off" by default so this line can be omitted.

Reply:
updated

Reply:
default off
has warning: undefined symbol off:
So I'm not sure this meaning

Reply:
Just remove the line as it is not needed.